samples/server/petstore/jaxrs-spec/swagger.json

```diff
@@ -760,24 +760,24 @@
           "description" : "None",
           "required" : true,
           "type" : "number",
-          "maximum" : 543.2,
-          "minimum" : 32.1
+          "maximum" : 543.200000000000045474735088646411895751953125,
+          "minimum" : 32.10000000000000142108547152020037174224853515625
         }, {
           "name" : "float",
           "in" : "formData",
           "description" : "None",
           "required" : false,
           "type" : "number",
-          "maximum" : 987.6,
+          "maximum" : 987.6000000000000227373675443232059478759765625,
```

Comment:
I'm on windows so I had to reassmble the shellscript, but in fact that shouldn't change the outcome. Anyway, please correct the shellscript and the samples, so we have a green state here again.